src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -0,0 +1,53 @@
+package org.testeditor.fixture.host;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testeditor.fixture.host.net.Connection;
+import org.testeditor.fixture.host.s3270.options.CharacterSet;
+import org.testeditor.fixture.host.s3270.options.TerminalMode;
+import org.testeditor.fixture.host.s3270.options.TerminalType;
+
+public class HostDriverFixture {
+  Connection con;
+
+  Logger logger = LoggerFactory.getLogger(HostDriverFixture.class);
+
+  /**
+   *
+   * @param s3270Path
+   * @param hostname
+   * @param port
+   */
+  public void connect(String s3270Path, String hostname, int port) {
```

Comment:
Method needs to be annotated with `@FixtureMethod` in order to be seen by the test-editor.